src/styles/card.scss

```diff
@@ -0,0 +1,86 @@
+.card {
+  background: #fff;
+  color: #{$primary-text-color};
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    display: block;
+    margin: 0 auto;
+    height: 134px;
+    width: 160px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 14px;
+    color: #{&secondary-text-color};
+    margin-bottom: 16px;
+  }
+
+  &__rate {
+    display: flex;
+    margin-bottom: 24px;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__counter {
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: 400;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+
+  &__price-title {
+    font-size: 12px;
+    line-height: 18px;
+    color: #{$secondary-text-color};
+  }
+
+  &__price-value {
+    font-size: 16px;
+    line-height: 18px;
+    font-weight: 700;
+  }
+
+  &__buy-button {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    box-sizing: border-box;
+    width: 100%;
+    height: 40px;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    border: 1px solid transparent;
+    border-radius: 5px;
+    text-transform: uppercase;
+    background: #{$blue-accent};
+    color: #fff;
+  }
+
+  &__buy-button:hover {
+    border-color: #{$blue-accent};
```

Comment:
no need to change border color on `hover`
you can move this color to your button styles